index.js

```diff
@@ -105,6 +98,30 @@ function loadAndValidateEnvironment (_opts) {
   return merge
 }
 
+function chooseAjvInstance (defaultInstance, ajvOpts) {
+  if (!ajvOpts) {
+    return defaultInstance
+  } else if (typeof ajvOpts === 'object' && typeof ajvOpts.customOptions === 'function') {
+    const ajv = ajvOpts.customOptions(getDefaultInstance())
+    if (!ajv) {
```

Comment:
```suggestion
    if (!(ajv instanceof Ajv)) {
```
Shouldn't this be better?